examples/src/main/proto/wfa/examples/streaming/hello_streaming_service.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.examples.streaming;
+
+option java_package = "org.wfanet.examples.streaming";
+option java_multiple_files = true;
+
+service HelloStreamingService {
+  rpc HelloStreaming(stream HelloStreamingRequest) returns (stream HelloStreamingResponse) {}
```

Comment:
nit: I think it's fine to stick with unidirectional streaming to avoid complication. I'd do client streaming to emulate the case of uploading a sketch.

Reply:
Done